### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by nayuki**
I enjoyed reading the fine articles on your site. Thank you for your hard work.

### CODE REVIEW

**Comment by nayuki** [comment]
Hello, any interest in these changes?

**Comment by Pomax** [comment]
Sorry, apparently gh never bothered to show me this issue existed. typo fixes are already appreciated!

File: gh-weblog-2/content/posts/markdown/2014-09-20-01-21-59.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-You run an open source project, and I have a meaningful contribution for you. I'd love to submit it, but now you're asking me for my personal information, without ever having shown that you can be trusted with that: I'm not signing your CLA.
+You run an open source project, and I have a meaningful contribution for you. I'd love to submit it, but now you're asking me for my personal information, without ever having shown that you can be trusted with that: I'm not signing your CLA (Contributor License Agreement).
```
Comment (Pomax) [inline gh-weblog-2/content/posts/markdown/2014-09-20-01-21-59.md]: I think I'd like to keep this as just CLA. If that raises questions, and someone has to google it, that's the kind of exercise that has meaning for the person doing the looking-up.

**Review by Pomax — CHANGES_REQUESTED** [review]
nice, just one request.

**Comment by nayuki** [comment]
Thanks for responding! I leave the changes in your hands now - you can do anything you like. I consider my work to be done.

**Review by Pomax — APPROVED** [review]

**Comment by Pomax** [comment]
thanks for the fixes!

**Pull request merged**